### PR TITLE
Temporary version of the Adding a Windows Node guide

### PR DESCRIPTION
**Pull request by fabi200123**
**Reason for PR**:
Adds a temporary version of the guide and the needed files to be able to add a Windows node to the kubernetes cluster until the main PR is merged.

Issue #
The guide PR still not merged!

**Notes**:
The main PR is [#239](https://github.com/kubernetes-sigs/sig-windows-tools/pull/239)

### CODE REVIEW

File: hostprocess/calico/calico.yml [hunk 1]
```diff
@@ -131,7 +131,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: sigwindowstools/calico-install:v3.20.0-hostprocess
+          image: sigwindowstools/calico-install:v3.24.5-hostprocess
```
Comment (aravindhp) [inline hostprocess/calico/calico.yml:134]: Shouldn't you be using `VERSION` here and elsewhere in this YAML?

File: hostprocess/flannel/flanneld/start.ps1 [hunk 2]
```diff
@@ -2,7 +2,8 @@ $ErrorActionPreference = "Stop";
 
 # flannel uses host-local, flannel.exe, and sdnoverlay so copy that to the correct location
 Write-Output "Copying SDN CNI binaries to host"
-Copy-Item -Path "$env:CONTAINER_SANDBOX_MOUNT_POINT/cni/*" -Destination "c:\opt\cni\bin" -Force
+mkdir -force $env:CNI_BIN_PATH
```
Comment (aravindhp) [inline hostprocess/flannel/flanneld/start.ps1:5]: Where does `CNI_BIN_PATH` get defined?

**Review by aravindhp — COMMENTED** [review]
Thanks for working on this, @fabi200123.

File: hostprocess/flannel/flanneld/start.ps1 (hunk 2 above)
Comment (fabi200123) [reply hostprocess/flannel/flanneld/start.ps1:5]: flannel-overlay has in it 
`- name: CNI_BIN_PATH`
     `value: C:\\Program Files\\containerd\\cni\\bin`
Same happens for kube-proxy and calico yml files

File: hostprocess/calico/calico.yml (hunk 1 above)
Comment (fabi200123) [reply hostprocess/calico/calico.yml:134]: Forgot to modify those too. I updated them now

**Comment by marosset** [comment]
overall looks good.

Can we replace `VERSION` in the files with `CALICO_VERSION`, `KUBE_PROXY_VERSION`, and `FLANNEL_VERSION`?
I think we have 3 different versions people need to replace for their deployments it will be easy to accidently replace the wrong with in bash.

File: guides/guide-for-adding-windows-node.md [hunk 3]
```diff
@@ -138,7 +194,8 @@ Use the command that was given to you when you ran `kubeadm init` on a control p
 
 For more information about it : https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/
 
-#### Verifying your installation
+#### Verifying your installation for flannel
```
Comment (marosset) [inline guides/guide-for-adding-windows-node.md:197]:
```suggestion
#### Verifying your installation for Flannel
```

**Review by marosset — APPROVED** [review]
/lgtm

Thanks for all this work @fabi200123!

**Pull request merged**